examples/api/custom_projection_example.py

```diff
@@ -1,3 +1,13 @@
+"""
+=================
+Custom projection
+=================
+
+This example showcases the Hammer projection by alleviating many features of
```

Comment:
I don't think alleviating is the right verb here; not sure what it's trying to say (utilizing/using would fit without changing the sentence, but doesn't quite imply the same things.)